janusgraph-core/src/main/java/org/janusgraph/graphdb/database/serialize/attribute/JsonSerializer.java

```diff
@@ -0,0 +1,67 @@
+package org.janusgraph.graphdb.database.serialize.attribute;
+
+import org.apache.tinkerpop.shaded.jackson.databind.JsonNode;
+import org.apache.tinkerpop.shaded.jackson.databind.ObjectMapper;
+import org.janusgraph.diskstorage.ScanBuffer;
+import org.janusgraph.diskstorage.WriteBuffer;
+import org.janusgraph.graphdb.database.serialize.OrderPreservingSerializer;
+
+import java.io.IOException;
+
+public class JsonSerializer<TYPE extends JsonNode> implements OrderPreservingSerializer<TYPE> {
```

Comment:
does JG have a precedent for multiple-letter template variables? if not, could we use T instead of TYPE please?

Reply:
Thank you for the review! I'm not sure if JG have a precedent for multiple-letter template variables but I will change TYPE to T.